### PR TITLE
Remove old yml file and made event name (type, topic, and subtopic) l…

### PR DESCRIPTION
**Pull request by owww**
…ower case.

# Pull Request Description

### "After" Screenshot

# Associated Issue(s)

# How to test

# Pull Request Submitter Checklist
+ [ ] I have provided a description of what this PR changes
+ [ ] I have linked an associated issue
+ [ ] I have provided steps for how a tester can verify this PR works correctly
+ [ ] I have included automated tests (or a description of why not)
+ [ ] I have let issue stakeholders know that this PR is open and ready for their review

<details>
  <summary><b>Optional Submitter Items:</b></summary>

- [ ] I have added PR comments to interesting or potentially confusing parts of the code
- [ ] I have incremented the version number according to [semantic versioning](https://semver.org)
- [ ] I have met with issue stakeholders and personally walked them through changes
- [ ] I have created new GitHub issues for "TODO" lines added by this PR
- [ ] I have moved the associated issue to "in review" in Codetree

</details>

# Pull Request Reviewer Checklist
+ [ ] The PR Description sufficiently details changes
+ [ ] Issue(s) have been linked with this PR
+ [ ] Adequate testing steps are provided
+ [ ] There are automated tests included in the PR
+ [ ] I have pulled the source code and tested this PR locally
+ [ ] I have read through the code and provided one or more comments

<details>
  <summary><b>Optional Reviewer Items:</b></summary>

- [ ] If I am not the first reviewer, I have made my own checklist

</details>

### CODE REVIEW

File: src/config.py [hunk 1]
```diff
@@ -40,9 +40,9 @@ class postgres:
 assets_api_url = f"https://fe-proxy.{events_env}.blacksky.com/assets-api"
 
 event_semantic_version = '3'
-event_type = 'Indicators and Warnings'
-event_topic = 'Missile Launches'
-event_subtopic = 'NOTAMs'
+event_type = 'indicators and warnings'
+event_topic = 'missile launches'
+event_subtopic = 'notams'
```
Comment (owrcasstevens) [inline src/config.py:45]: make sure the last two are includes as arrays

File: cloudformation/notam-launch.yml [hunk 2]
```diff
@@ -1,254 +0,0 @@
...
-      Type: String
-      Value: 'batch'
-
-  DeploymentPackageNameParameter:
-    Type: "AWS::SSM::Parameter"
-    Properties:
-      Name: !Sub '/env/${EnvironmentName}/deployment/${ServiceName}/deployment_package_name'
-      Description: "File name of deployment package"
-      Type: String
-      Value: !Ref 'DeploymentPackageName'
```
Comment (owrcasstevens) [inline cloudformation/notam-launch.yml:254]: why did this get removed?

Comment (owww) [reply cloudformation/notam-launch.yml:254]: Sean wanted to hold off on SIT and EXPO until DEV looks good

File: src/config.py (hunk 1 above)
Comment (owww) [reply src/config.py:45]: Noted

**Pull request merged**